docs/paying-for-lit/creating-session-signatures.md

```diff
@@ -0,0 +1 @@
+# Creating Session Signatures
```

Comment:
I don't know if this section belongs here. Maybe can just mention payment is a pre-condition to creating SessionSigs in the payment overview, and link to our main sessionSigs/authentication pages (and vice versa - point back here to payment in the sessionSigs docs)